demo-private.html

```diff
@@ -4,9 +4,10 @@
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>google-sheets Demo</title>
-  <script src="../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../webcomponentsjs/webcomponents.js"></script>
```

Comment:
I was switching to the non-min since the `.min` file doesn't exist when developing/testing locally. Is there a better work around to not having to change this to test with a [local git checkout](https://www.polymer-project.org/0.5/resources/tooling-strategy.html#git) of all the components?